lua/entities/gmod_tardis/modules/sh_artronenergy.lua

```diff
@@ -0,0 +1,311 @@
+if SERVER then
+
+    local artron_values = {
+        cost_demat = -540,
+        cost_mat = -360,
+        cost_full = -1200,
+        cost_controls = {
+            ["cloak"] = -720,
+            ["physlock"] = -540,
+            ["float"] = -540,
+        },
+        cost_hads = -180,
+        cost_failed_demat = -80,
+
+        -- every 1 second:
+        spend_vortex_teleport = -32,
+        spend_flight = -45,
+        spend_flight_boost = -45,
+        spend_cloak = -36,
+        spend_cloak_handbrake = -12,
+
+        -- every 5 seconds:
+        increase_handbrake = 180 * 5,
```

Comment:
This function is fine but it seems almost necessary as otherwise charging is quite slow and we do not tell the users about this at all - should we consider some kind of hint when charging and energy is low maybe? This links in with #176

Reply:
I've been thinking about that, but we'd need a whole separate thing for that
(I don't think hints would do the trick)

Reply:
I've made it so that artron would recharge every second with handbrake enabled, and still once in 5 seconds when it's disabled. That would make it more obvious for anyone who uses handbrake at least once